src/DataMapper/DataMapper.php

```diff
@@ -94,11 +96,38 @@ public function mapFormsToData($forms, &$data): void
                 } elseif ($writePropertyPath instanceof \Closure) {
                     $writePropertyPath($data, $form->getData());
                 } else {
-                    $this->propertyAccessor->setValue($data, $writePropertyPath, $form->getData());
+                    $writePropertyPaths[$writePropertyPath][] = $form;
                 }
             } catch (\Throwable $e) {
                 $this->formExceptionHandler->handleException($form, $data, $e);
             }
         }
+
+        foreach ($writePropertyPaths as $writePropertyPath => $forms) {
+            try {
+                if (1 === \count($forms)) {
+                    $this->propertyAccessor->setValue($data, $writePropertyPath, reset($forms)->getData());
+                } else {
+                    $formData = [];
+
+                    foreach ($forms as $form) {
+                        $formData[$form->getName()] = $form->getData();
+                    }
+
+                    $method = new \ReflectionMethod(\get_class($data), $writePropertyPath);
+                    $arguments = [];
+
+                    foreach ($method->getParameters() as $parameter) {
+                        $arguments[] = $formData[$parameter->getName()];
+                    }
+
+                    ([$data, $writePropertyPath])(...$arguments);
```

Comment:
dann ist nur noch ein methoden-name als `writePropertyPath` zulässig?

Reply:
Yes, that's a limitation right now. But I'd rather limit the feature like this for a first attempt. Making this more flexible in the future is still possible.